engine/src/flutter/shell/platform/common/accessibility_bridge_unittests.cc

```diff
@@ -209,9 +209,15 @@ TEST(AccessibilityBridgeTest, CanHandleSelectionChangeCorrectly) {
   std::shared_ptr<TestAccessibilityBridge> bridge =
       std::make_shared<TestAccessibilityBridge>();
   FlutterSemanticsNode2 root = CreateSemanticsNode(0, "root");
-  root.flags = static_cast<FlutterSemanticsFlag>(
-      FlutterSemanticsFlag::kFlutterSemanticsFlagIsTextField |
-      FlutterSemanticsFlag::kFlutterSemanticsFlagIsFocused);
+
+  auto flags = FlutterSemanticsFlags{
+
+      .is_text_field = true,
+      .is_read_only = true,
+  };
```

Comment:
I believe this matches our style guide:

```suggestion
  FlutterSemanticsFlags flags = {
      .is_text_field = true,
      .is_read_only = true,
  };
```